libs/model/src/community/CreateCommunity.command.ts

```diff
@@ -218,6 +111,24 @@ export function bech32ToHex(address: string) {
   }
 }
 
+function baseToNetwork(n: ChainBase): ChainNetwork {
+  switch (n) {
+    case ChainBase.CosmosSDK:
+      return ChainNetwork.Osmosis;
+    case ChainBase.Substrate:
+      return ChainNetwork.Edgeware;
+    case ChainBase.Ethereum:
+      return ChainNetwork.Ethereum;
+    case ChainBase.NEAR:
+      return ChainNetwork.NEAR;
+    case ChainBase.Solana:
+      return ChainNetwork.Solana;
+    default:
+      // @ts-expect-error <StrictNullChecks/>
```

Comment:
Adding ChainNetwork | null to the return type doesn't work?

Reply:
I didn't write this function; I just moved it. Either way returning null should not/will not happen so we will have to assert not null somewhere. Will handle tomorrow.